### PR TITLE
">" changed to "-"

### PR DESCRIPTION
**Pull request by aosyatnik**
If I got the idea rightly... Then here is the change.

Reviewers
@Benglin ,@vmoyseenko, please take a look.

Link to YouTrack:
[MAGN-6058](http://adsk-oss.myjetbrains.com/youtrack/issue/MAGN-6058) DUI: Change ‘>’ to ‘–‘ for breadcrumbs in search results

### CODE REVIEW

**Comment by aosyatnik** [comment]
I've added to this PR 1 more task. 
[MAGN-5559](http://adsk-oss.myjetbrains.com/youtrack/issue/MAGN-5559) DUI: Update margins

The member highlight size needs to be 32px -- right now this is right. But there is a margin indicated as green color → remove that
image

File: src/DynamoCoreWpf/Views/Search/LibraryView.xaml [hunk 1]
```diff
@@ -660,11 +660,38 @@
 
                                         <Border Grid.Row="1"
                                                 Background="#2F2F2F">
-                                            <ItemsPresenter x:Name="ItemsHost"
-                                                            Margin="0,3,0,3">
+                                            <ItemsPresenter x:Name="ItemsHost">
```
Comment (Benglin) [inline src/DynamoCoreWpf/Views/Search/LibraryView.xaml]: Can we not use a converter here for the `Margin`? Maybe make a new one that is called `NodeTypeToMarginConverter` that will give the right margin size based on the item type. So this whole thing can be simplified as:

```xml
<ItemsPresenter x:Name="ItemsHost">
    <ItemsPresenter.Margin>
        <Binding Path="..."
                 Converter="{StaticResource NodeTypeToMarginConverter}" />
    </ItemsPresenter.Margin>
</ItemsPresenter>
```

Is this possible?

**Comment by aosyatnik** [comment]
Hi, @Benglin . There is no need to create new converter... I found suitable.

**Comment by Benglin** [comment]
Nice, thanks @aosyatnik! LGTM.

**Pull request merged**